extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -575,16 +576,16 @@ export async function promptConnectionStringPasswordAndUpdateConnectionString(co
 				connectionString = await vscodeMssqlApi.getConnectionString(connectionDetails, true, false);
 			}
 		}
-		// set connection string to not include the password if connection info does not include password, or user chooses to not include password, or authentication type is not sql login
-		let userPassword: string | undefined;
-		if (includePassword !== constants.yesString) {
+
+		if (includePassword !== constants.yesString || !connectionInfo.password || connectionInfo.authenticationType !== 'SqlLogin') {
+			// set connection string to not include the password if connection info does not include password,
```

Comment:
`get connection string without the password...` is a bit more accurate since we're not really "setting" it